apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1480,8 +1505,12 @@ fn handle_error<T: Into<opentelemetry::global::Error>>(err: T) {
 register_plugin!("apollo", "telemetry", Telemetry);
 
 fn request_ftv1(mut req: SubgraphRequest) -> SubgraphRequest {
-    if req.context.contains_key(ENABLE_SUBGRAPH_FTV1)
-        && Span::current().context().span().span_context().is_sampled()
+    if req
+        .context
+        .private_entries
+        .lock()
+        .contains_key::<EnableSubgraphFtv1>()
+        && !Span::current().is_disabled()
```

Comment:
I just want to make sure we have manually checked that this functionality is correct:
```!Span::current().is_disabled()```
Run the router with 0.5 global tracing and 0.25 ftv1. In Apollo studio the traces with ftv1 should come in approx one quarter of the time.

Reply:
I validated with a manual test that it measures FTV1 at the same ratio as dev (here, that would be 0.5). I added an explanation of the ratios in https://github.com/apollographql/router/pull/3072, since it was still a bit confusing to me